lastuser_core/models/user.py

```diff
@@ -309,7 +310,9 @@ def _set_password(self, password):
         if password is None:
             self.pw_hash = None
         else:
-            self.pw_hash = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+            self.pw_hash = bcrypt.hashpw(
+                password.encode('utf-8'), bcrypt.gensalt()
+            ).decode('utf-8')
```

Comment:
This part needs to be tested. Does the bcrypt module expect bytes instead of a string, as implied here?

Reply:
Yes. I tested this.

Reply:
brypt always expected bytes, hence we were encoding the input even before. what changed is that it started returning bytes instead of string, hence added the decode.

Reply:
But the return is not UTF-8. The db column has to be changed to a byte array (`bytea` I think).

Reply:
Or at least decode it as ASCII instead of UTF-8.

Reply:
Confirmed. Use `.decode('ascii')`

Reply:
Ok. Done.